src/common/Parameter.cpp

```diff
@@ -2814,6 +2821,71 @@ void Parameter::get_display_of_modulation_depth(char *txt, float modulationDepth
 
         break;
     }
+    case ct_freq_shift:
```

Comment:
Why do we need this code and below rather than just a linear scale with extend config above?

Reply:
It already has the correct extend config (100x) but the parameter base range is +/-10. And so the DSP did a wrong thing where instead of get_extended it manually applied another 10 or 1000 multiply, which created the too wide range and incorrect stringification.

Reply:
I mean I would love it if there's a way to have no code copypasta I just didn't see it immediately...

Reply:
I guess my question is : "How is this freq shift just different than a linear param, so can we delete all this code". It just looks like it's a linear with a scale factor of 10, which we can do without a special case. Or am I misreading?

Reply:
Right it is linear with a scale factor of 10 and then when extended a scale factor 100x that. So you reckon this would work with the default Linear type?

Reply:
OK this should be resolved now. There is displayInfo.scale so I just set it to 10 for ct_freq_shift and that seems to work!